src/Components/CreateImageWizard/formComponents/Oscap.tsx

```diff
@@ -17,37 +19,47 @@ import {
   SelectVariant,
 } from '@patternfly/react-core/deprecated';
 import { HelpIcon } from '@patternfly/react-icons';
-import PropTypes from 'prop-types';
 
 import OscapProfileInformation from './OscapProfileInformation';
 
 import {
+  DistributionProfileItem,
   useGetOscapCustomizationsQuery,
   useGetOscapProfilesQuery,
 } from '../../../store/imageBuilderApi';
 import { reinitFileSystemConfiguratioStep } from '../steps/fileSystemConfiguration';
 import { reinitPackagesStep } from '../steps/packages';
 
+// eslint-disable-next-line
+type ChangeType = <F extends keyof Record<string, any>>(
+  name: F,
+  // eslint-disable-next-line
```

Comment:
Why do you need to disable the linter here?

Reply:
because there was error  when runnig the lint: 33:51  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any
  35:26  error  Unexpected any. Specify a different type  @typescript-eslint/no-explicit-any

Reply:
Thanks for the explanation. Unfortunately you can't disable the linter in that case. You need to find the correct type to replace `any`.